include/deal.II/lac/scalapack.h

```diff
@@ -199,8 +200,12 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
    * internally the distributed matrix is copied to one process, which
    * does the output. Therefore, the matrix has to fit into the memory
    * of one process.
+   *
+   * To tweak the I/O performance, especially for parallel I/O, the user may define the optional parameter @p chunk_size.
```

Comment:
can you please elaborate what the two numbers in `std::pair` mean and what are the valid values? If these are always ranges of rows, then there must be an assert on their values. Also from documentation I was not sure whether this input should be the same on all MPI cores or different.